tensornetwork/backends/numpy/numpy_backend_test.py

```diff
@@ -75,10 +75,10 @@ def test_sqrt():
 
 def test_diag():
   backend = numpy_backend.NumPyBackend()
-  a = backend.convert_to_tensor(np.array([[1, 2, 3], [4, 5, 6]]))
+  a = backend.convert_to_tensor(np.array([[1.0, 2, 3], [4, 5, 6]]))
```

Comment:
Please make all of these values floats instead of just one.

Reply:
Done